src/MapGT.js

```diff
@@ -17,11 +23,14 @@ export default class MapGT {
             if (!this._mapDOM) {
                 console.warn(`contentDocument of the SVG is null and SVG DOM manipulation will not be possible. Try setting up a simple server to by pass the CORS issue.
     Suggested solution for dev: python -m http.server`);
+                return;
             }
-            window.area = this._mapDOM.getElementById("1");
-            const area = new Area(window.area, 1);
-            console.log(area);
-            area.highlight();
+            this.views = [];
+            this.areas = [];
+            this._populateViews();
+            this.currentView = Array.from(this._mapDOM.querySelectorAll(".view"))
+                .filter(el => el.attributes.visibility.nodeValue == "visible")[0];
```

Comment:
I would just initialize the first view as visible and flick the others off - it'd be a weird experience to not default to first.